nuxeo-distribution/nuxeo-distribution-resources/src/main/resources/templates-tomcat/common-base/client/scripts/upgrade_tomcat7.sh

```diff
@@ -27,13 +27,15 @@ readonly TOMCAT_LATEST_URL="http://www.apache.org/dist/tomcat/tomcat-7"
 
 usage() {
   echo "Usage:"
-  echo -e "\t./$(basename "$0") NUXEO_HOME [TOMCAT_TARGET_VERSION]"
+  echo -e "\\t./$(basename "$0") NUXEO_HOME [TOMCAT_TARGET_VERSION]"
   echo
   echo "Note: If no target version is specified the latest one will be retrieved from the Tomcat site"
   echo
+  echo "Warning: this script cannot be run with sudo -c. The Nuxeo user has to be logged in."
+  echo
   echo "Examples:"
-  echo -e "\t./$(basename "$0") /path/to/nuxeo-cap-7.10-tomcat"
-  echo -e "\t./$(basename "$0") /path/to/nuxeo-cap-7.10-tomcat 7.0.76"
+  echo -e "\\t./$(basename "$0") /path/to/nuxeo-cap-7.10-tomcat"
+  echo -e "\\t./$(basename "$0") /path/to/nuxeo-cap-7.10-tomcat 7.0.76"
```

Comment:
This is a cleanup and should be a separate commit from the one fixing the basic RHEL/CentOS problem.